kubernetes/opentelemetry-demo.yaml

```diff
@@ -7148,6 +7148,8 @@ spec:
             value: http://$(OTEL_COLLECTOR_NAME):4317
           - name: OTEL_RESOURCE_ATTRIBUTES
             value: service.name=$(OTEL_SERVICE_NAME),service.instance.id=$(OTEL_K8S_POD_UID),service.namespace=opentelemetry-demo,k8s.namespace.name=$(OTEL_K8S_NAMESPACE),k8s.node.name=$(OTEL_K8S_NODE_NAME),k8s.pod.name=$(OTEL_K8S_POD_NAME)
+          - name: DOTNET_HOSTBUILDER__RELOADCONFIGONCHANGE
```

Comment:
Would it be helpful to add a comment or link to the issue?

https://github.com/dotnet/AspNetCore.Docs/issues/19814

Reply:
Done, thanks!